src/server/dashboard-controller.js

```diff
@@ -24,11 +24,14 @@ class UserController {
     if (this.serverConfig.areRoomsEnabled) {
       rooms = await this.roomService.getRoomsOwnedOrJoinedByUser(user._id);
     }
-    const mappedRooms = await Promise.all(rooms.map(room => this.clientDataMappingService.mapRoom(room, user)));
     const activities = await this.dashboardService.getUserActivities({ userId: user._id, limit: 10 });
+    const favorites = await this.dashboardService.getUserFavorites(user._id);
```

Comment:
I don't think we have to fetch them from the DB, they should be there by `user.favorites` (as the `req.user` is the DB user object).